frontend/webpack.config.ts

```diff
@@ -34,7 +35,9 @@ const WDS_PORT = 8080;
 /* Helpers */
 const extractCSS = new MiniCssExtractPlugin({
   filename:
-    NODE_ENV === 'production' ? 'app-bundle.[contenthash].css' : 'app-bundle.[name].[hash].css',
+    NODE_ENV === 'production'
+      ? 'app-bundle.[name].[contenthash].css'
```

Comment:
Need the `[name]` to identify later when we want to exclude certain chunks from the HTML